.eslintrc.js

```diff
@@ -285,6 +285,10 @@ module.exports = {
     'webex-core.js',
     'ediscovery.js',
     'CHANGELOG.md',
+    'README.md',
+    'LICENSE.md',
+    'package.json',
+    'yarn.lock',
```

Comment:
Was getting interesting ESLint issue over this, in which it was unable to validate these files.

If anyone knows a better solution, I'm welcome to making the changes necessary.